Tetrahedral_remeshing/doc/Tetrahedral_remeshing/dependencies

```diff
@@ -8,3 +8,4 @@ Triangulation_3
 BGL
 Mesh_3
 TDS_3
+Surface_mesh
```

Comment:
why?

Reply:
because one of the examples with Mesh_3 uses a `Surface_mesh` as input